### ISSUE TITLE
Add tooltips to clickable elements and alt text to images

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#981]
In order to improve accessibility we want to add tooltips to all anchors, buttons, md-buttons and md-tabs. Tooltips should be used as aria-labels as well (automatically for md-button, needs to be checked for ordinary anchors, buttons and md-tabs).
Additionally all images (img) will get an alt text.

**Comment by rreitmann** [comment]
we won't adjust the administration dialogs...

**Comment by rreitmann** [comment]
@limouri: here are some things left todo:
- [ ] rework the tooltips of the toolbar header
- [ ] change tooltip texts of submenu in navbar when open

**Comment by rreitmann** [comment]
@limouri One additional thing:
- [ ] The bmbf logo should be an image with an alt text not a css background image

**Comment by rreitmann** [comment]
@KeLange Please review the tooltips on all browsers in all languages

**Comment by KeLange** [comment]
- In the IE the tooltips for the tabs remain visible after clicking on a tab.
- The english tooltips for the tabs contain capital letters.
- In Firefox: some buttons have a tooltip, but are not clickable:
       - The upload button if no data project is selected
       - The button "Show complete distribution", although some variables exist where the table of frequencies is completely shown  (e.g. var-abs2005-ds1-bdem11_g1!)
       - The three buttons delete, validate, release data acquisation project.
- The english button for the survey upload is: "Click to upload upload surveys...".

**Comment by rreitmann** [comment]
There are hardcoded tooltips with text "Tooltip" in:
- [ ] create-project-dialog.html.tmpl
- [ ] release-project-dialog.html.tmpl
